kong/plugins/influxdb-log/handler.lua

```diff
@@ -0,0 +1,105 @@
+local HttpLogHandler = require "kong.plugins.http-log.handler"
+
+local InfluxdbLogHandler = HttpLogHandler:extend()
```

Comment:
Instead of extending another plugin (HttpLog), can't you just use:

```lua
local basic_serializer = require "kong.plugins.log-serializers.basic"
return cjson.encode(basic_serializer.serialize(ngx))
```